packages/content/templates/nuxt-content.js

```diff
@@ -5,6 +5,7 @@ const rootKeys = ['class-name', 'class', 'style']
 const rxOn = /^@|^v-on:/
 const rxBind = /^:|^v-bind:/
 const rxModel = /^v-model/
+const rxClass = /^(?::|v-bind:)*class/
```

Comment:
These 3 cases will throw an error due to limitation of Nuxt Content.

```html
<my-component v-bind:class="{ 'my-class': true }">
<my-component v-bind:class="[ 'my-class' ]">
<my-component v-bind:class="'my-class'">
```

Should we only detect just `"class"` attribute?